### ISSUE TITLE
Website down

### ISSUE THREAD

**Issue by codemaster138** [bleenco/abstruse#566]
The website is invariable timing out for me. I can't speak for the rest of the world, but at least in Germany, your website seems to be completely gone...

**Comment by axeloz** [comment]
Hello
I just installed Abstruse yesterday and it looks awesome. But it looks very abandoned too. There are many contributors on this project, how comes none is communicating about the project's status? 
Can we still rely on Abstruse?
Thanks

**Comment by jkuri** [comment]
Hi @axeloz, its true, project is more or less abandoned, I do not work at bleenco anymore but still have abstruse.cc domain though, so if there is anyone ready to host demo I can point domain to that ip.

**Comment by axeloz** [comment]
Thanks for your lightning fast answer. So no plan on keep this project alive? What is missing the most to me is the documentation.

**Comment by jkuri** [comment]
documentation was never the highlight of this project to be honest ;) you can check the https://github.com/bleenco/abstruse/tree/master/docs folder, but I believe you did this already.

**Comment by kbingham** [comment]
That's a shame, abstruse has one of the nicest/cleanest UI's I've seen around and was hoping to make use of it with gitea for our project.

**Comment by axeloz** [comment]
> @kbingham it works with gitea btw.

Yes! That’s why I installed it, I love Gitea I am using it even before the fork of gogs. 

Yes I did check the documentation but I find it quite light. I could not start a successful build and deployment so far. I kind of gave up and switched to DroneCI (could not run a build either yet). 

As abstruse is coded in Go we should maybe talk to the Gitea team. They could be interested in taking over. I would make a lot of sense for them.
